.github/workflows/analysis.yml

```diff
@@ -0,0 +1,109 @@
+name: Analysis
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    types: [opened, reopened, synchronize, ready_for_review, converted_to_draft]
+  schedule:
+    - cron: "0 11 * * 0" # 3 AM PST = 12 PM UDT, runs sundays
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  conventional-commit-check:
+    name: conventional-commit-check
+    if: ${{ ! github.event.pull_request.draft }}
+    runs-on: ubuntu-latest  # Runs on the latest version of Ubuntu
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4  # Checks out the code for the current pull request
+        with:
+          fetch-depth: 0
+
+      - name: Check commit messages
```

Comment:
Suggestion only!  Hand off convention commit handling to the qs-helpers' pr-validate?  It'll also do PR description changes.

https://github.com/bcgov/quickstart-openshift/blob/main/.github/workflows/pr-validate.yml#L14